frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -1,30 +1,120 @@
+                    >
+                        docs
+                    </Link>
+                    .
+                </p>
+            </div>
+            <div className="space-y-2">
+                <LemonSwitch
+                    data-attr="opt-in-capture-console-log-switch"
+                    onChange={(checked) => {
+                        updateCurrentTeam({ capture_console_log_opt_in: checked })
+                    }}
+                    label="Capture console logs"
+                    bordered
+                    checked={currentTeam?.session_recording_opt_in ? !!currentTeam?.capture_console_log_opt_in : false}
+                    disabled={!currentTeam?.session_recording_opt_in}
+                />
+                <p>
+                    This setting controls if browser console logs will be captured as a part of recordings. The console
+                    logs will be shown in the recording player to help you debug any issues.
+                </p>
+            </div>
+            <div className="space-y-2">
+                <LemonSwitch
+                    data-attr="opt-in-capture-performance-switch"
+                    onChange={(checked) => {
+                        updateCurrentTeam({ capture_performance_opt_in: checked })
+                    }}
+                    label="Capture network performance"
+                    bordered
+                    checked={currentTeam?.session_recording_opt_in ? !!currentTeam?.capture_performance_opt_in : false}
+                    disabled={!currentTeam?.session_recording_opt_in}
+                />
+                <p>
+                    This setting controls if performance and network information will be captured alongside recordings.
+                    The network requests and timings will be shown in the recording player to help you debug any issues.
+                </p>
+            </div>
+        </div>
+    )
 }
 
-function ReplayCostControl(): JSX.Element {
+export function ReplayAuthorizedDomains(): JSX.Element {
+    return (
+        <div className="space-y-2">
+            <p>
+                Use the settings below to restrict the domains where recordings will be captured. If no domains are
+                selected, then there will be no domain restriction.
+            </p>
+            <p>
+                Domains and wildcard subdomains are allowed (e.g. <code>https://*.example.com</code>). However,
+                wildcarded top-level domains cannot be used (for security reasons).
+            </p>
+            <AuthorizedUrlList type={AuthorizedUrlListType.RECORDING_DOMAINS} />
+        </div>
+    )
+}
+
+export function ReplayCostControl(): JSX.Element {
     const { updateCurrentTeam } = useActions(teamLogic)
     const { currentTeam } = useValues(teamLogic)
 
     return (
         <FlaggedFeature flag={FEATURE_FLAGS.SESSION_RECORDING_SAMPLING}>
             <>
-                <h2 className={'flex flex-row align-center space-x-2'}>
```

Comment:
Kill?

Reply:
yup -sorry I should have done a scan of this myself.